Update simpletest with 8.0 standards

DJDevon3 opened the pull request:
Includes displayio initiatlization missing from outdated example. This was an issue dealing with support questions related to missing displayio import. Also added height and width ints to make it easier for beginners to change screen sizes as these OLED displays come in a variety of sizes now.

DJDevon3 committed:
Update simpletest with 8.0 standards

Includes displayio initiatlization missing from outdated example. This was an issue dealing with support questions related to missing displayio import. Also added height and width ints to make it easier for beginners to change screen sizes as these OLED displays come in a variety of sizes now.

DJDevon3 commented:
@kattni [Learn Guide for the Monochrome I2C OLED display](https://learn.adafruit.com/monochrome-oled-breakouts/circuitpython-usage) appears to have a much more current example in the learn guide than the simpletest example. Learn guide doesn't need to be updated, the simpletest example needed to be brought up to the learn guide standard not the other way around. :)

kattni commented:
@DJDevon3 Thank so much for looking into the guide! 

It looks like the CI is failing because there are two unused imports after your update. Please remove the imports and commit again to this PR. Thanks!

DJDevon3 committed:
fixed unused time import

pylint complained about the unused import. nice catch pylint!

DJDevon3 committed:
removed busio import

Not sure if this is the right thing to do if the import is part of the optional commented out section.  Guess someone will cross that bridge when they get there?

kattni commented:
@evaherrada Please review and complete any guide updates required.

evaherrada reviewed (approved):
Looks good to me. Merging
